src/libraries/System.Text.Json/tests/System.Text.Json.FSharp.Tests/EnumTests.fs

```diff
@@ -15,6 +15,13 @@ type BadEnum =
 let badEnum = BadEnum.``There's a comma, in my name`` ||| BadEnum.``There's a comma, even here``
 let badEnumJsonStr = $"\"{badEnum}\""
 
+type NumericLabelEnum =
```

Comment:
Nit: could you group the type definition with the tests that use it?

Reply:
Fixed.